armi/reactor/converters/geometryConverters.py

```diff
@@ -560,6 +564,15 @@ def _setupConvertedReactor(self, grid):
         self.convReactor.core.p.power = self._sourceReactor.core.p.power
         self.convReactor.core.name += " - {0}".format(self._GEOMETRY_TYPE)
 
+        # It is important to map the nuclide categories from the source
+        # reactor so that the fuel, coolant, and structures are mapped to
+        # the same nuclides. This comes into play in areas of the framework
+        # (like in lattice physics), where the categorization makes an impact
+        # for physics modeling.
+        self.convReactor.core._nuclideCategories = (
+            self._sourceReactor.core._nuclideCategories
+        )
```

Comment:
The fact that it's listed as a `_private` variable is a bit broken with this external usage. Given this, I guess it probably shouldn't be private, or it should be accessed via a setter.